<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Rasheed-pc" name="com.truaxis.reporting.test.Scenario5Test" tests="24" failures="1" timestamp="6 Dec 2013 20:16:48 GMT" time="576.597" errors="3">
  <testcase name="testRD_5_3" time="0.047" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_22" time="20.071" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_2" time="1.685" classname="com.truaxis.reporting.test.Scenario5Test">
    <error type="java.lang.AssertionError" message="Export,PDF,Exel,CSV labels,are not displayed on Coupon Rewards Page expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Export,PDF,Exel,CSV labels,are not displayed on Coupon Rewards Page expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at com.truaxis.reporting.test.Scenario5Test.testRD_5_2(Scenario5Test.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testRD_5_2 -->
  <testcase name="testRD_5_12" time="1.123" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_5" time="0.078" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_6" time="0.031" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_8" time="0.437" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_19" time="88.935" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_21" time="89.852" classname="com.truaxis.reporting.test.Scenario5Test">
    <failure type="org.openqa.selenium.TimeoutException" message="Timed out after 30 seconds waiting for com.core.actionhandlers.WaitForActionsHandler$WaitForWebElements$3@1b380c
Build info: version: &#039;2.35.0&#039;, revision: &#039;8df0c6bedf70ff9f22c647788f9fe9c8d22210e2&#039;, time: &#039;2013-08-17 12:46:41&#039;
System info: os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.7.0&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for com.core.actionhandlers.WaitForActionsHandler$WaitForWebElements$3@1b380c
Build info: version: '2.35.0', revision: '8df0c6bedf70ff9f22c647788f9fe9c8d22210e2', time: '2013-08-17 12:46:41'
System info: os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:259)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at com.core.actionhandlers.WaitForActionsHandler$WaitForWebElements.exists(WaitForActionsHandler.java:78)
	at com.truaxis.common.PageBase.genericGetWebElement(PageBase.java:130)
	at com.truaxis.common.PageBase.returnWebElement(PageBase.java:61)
	at com.truaxis.common.PageBase.getText(PageBase.java:198)
	at com.truaxis.reporting.pageobjects.RDHomePage.verifytheMerchantHeaderAfterUpdatefilterClick(RDHomePage.java:618)
	at com.truaxis.reporting.test.Scenario5Test.testRD_5_21(Scenario5Test.java:252)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- testRD_5_21 -->
  <testcase name="testRD_5_4" time="0.109" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_10" time="0.390" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_13" time="7.972" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_11" time="0.702" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_20" time="91.323" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_17" time="52.119" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_18" time="53.696" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_16" time="50.591" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_14" time="29.453" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_7" time="30.389" classname="com.truaxis.reporting.test.Scenario5Test">
    <error type="java.lang.AssertionError" message="Merchant Filter DropDown Labels,is not displayed on Coupon Rewards Page expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Merchant Filter DropDown Labels,is not displayed on Coupon Rewards Page expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at com.truaxis.reporting.test.Scenario5Test.testRD_5_7(Scenario5Test.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testRD_5_7 -->
  <testcase name="testRD_5_15" time="33.041" classname="com.truaxis.reporting.test.Scenario5Test">
    <error type="java.lang.AssertionError" message="Failed either because 1.MerchantName not available 2.Update filter button dint display any result  expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed either because 1.MerchantName not available 2.Update filter button dint display any result  expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.truaxis.reporting.test.Scenario5Test.testRD_5_15(Scenario5Test.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testRD_5_15 -->
  <testcase name="testRD_5_9" time="0.421" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_24" time="5.136" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_23" time="18.950" classname="com.truaxis.reporting.test.Scenario5Test"/>
  <testcase name="testRD_5_1" time="0.046" classname="com.truaxis.reporting.test.Scenario5Test"/>
</testsuite> <!-- com.truaxis.reporting.test.Scenario5Test -->
